linkerd/app/inbound/src/policy/http/tests.rs

```diff
@@ -1,6 +1,7 @@
 use super::*;
 use crate::policy::{Authentication, Authorization, Meta, Protocol, ServerPolicy};
 use linkerd_app_core::{svc::Service, Infallible};
+use linkerd_proxy_server_policy::local_rate_limit::LocalRateLimit;
```

Comment:
With a reexport we can simplify this:
```suggestion
use linkerd_proxy_server_policy::LocalRateLimit;
```